core/shared/src/main/scala/io/circe/Decoder.scala

```diff
@@ -617,6 +619,10 @@ final object Decoder extends TupleDecoders with LowPriorityDecoders {
     override final def product[A, B](fa: Decoder[A], fb: Decoder[B]): Decoder[(A, B)] = fa.and(fb)
     final def flatMap[A, B](fa: Decoder[A])(f: A => Decoder[B]): Decoder[B] = fa.flatMap(f)
   }
+
+  implicit val decodeByteBuffer: Decoder[ByteBuffer] = decodeString.map { b64String =>
```

Comment:
I'd probably vote for using `decodeString.map(s => ....)` style (no curly braces) given that there is just one simple expression in the closure.